packages/chopsticks/src/plugins/set-runtime-log-level/index.ts

```diff
@@ -0,0 +1,14 @@
+import { Handler, ResponseError } from '../../rpc/shared'
+import { defaultLogger } from '../../logger'
+
+export const rpc: Handler = async (context, [runtimeLogLevel]) => {
+  defaultLogger.debug({ runtimeLogLevel }, 'dev_setRuntimeLogLevel')
+
+  if (typeof runtimeLogLevel !== 'number' || runtimeLogLevel < 0 || runtimeLogLevel > 5) {
```

Comment:
no need to check the value, leave it to user to set correct value because it depends on runtime which value is correct